ext/protobuf.mk

```diff
@@ -30,6 +30,9 @@ $(4)/protoc: $(if $(call sne,$(1),$(HOSTARCH)),$(HOSTBIN)/protoc)
 	&& ./autogen.sh > configure-log.txt 2>&1 \
 	&& TMP=$(PWD)/$(TMP) ./configure \
 		--prefix $(PWD)/$(BUILD)/$(1) \
+		--libdir=$(PWD)/$(2) \
```

Comment:
The problem with this is that it spams masses of unnecessary files into the docker build.  Anything that is needed but was missing should be copied into the tree after build by the rules below, rather than done like this.